cmd/client.go

```diff
@@ -33,15 +33,15 @@ func newClientCommand(ctx *cmdContext) *cobra.Command {
 			log.SetLogger("QedClient", ctx.logLevel)
 
 			clientCtx.client = client.NewHTTPClient(client.Config{
-				Endpoint: clientCtx.endpoint,
+				Cluster:  client.QEDCluster{Endpoints: clientCtx.endpoints, Leader: clientCtx.endpoints[0]},
```

Comment:
Change constructor to pass only a slice of endpoints or strings, and then, construct the QEDCluster struct after getting info about the leader node.

Reply:
Done!